src/components/Note/Note.tsx

```diff
@@ -105,14 +111,25 @@ export const Note = (props: NoteProps) => {
   const stackSetting: "stackOntop" | "stackBetween" | "stackBelow" = "stackBetween";
 
   return (
-    <div className={classNames("note__root")}>
+    <div className={classNames("note__root", getColorClassName(column?.color))}>
```

Comment:
You don't need to add `getColorClassName(column?.color)` here because the accent colors are already provided by the parent / column component. Since the column already has the specific class name for it's color, all the css properties belonging to this classname can also be used by all of the children of a column.